game/hud/src/services/session/layout.ts

```diff
@@ -86,20 +92,26 @@ export function saySomething(s: string) : LayoutAction {
   }
 }
 
+export function adjustWidgetPositions() {
+  return {
+    type: ADJUST_POSITIONS,
+  }
+}
 
 
-let defaultWidgets = () => {
+const defaultWidgets = (): any => {
   return {
-    'PlayerHealth': {x:100,y:300,scale:1,height:200,width:350},
-    'TargetHealth': {x:600,y:300,scale:1,height:200,width:350},
-    'Warband': {x:100,y:10,scale:1,height:200,width:350},
-    'Chat': {x:10,y:100,scale:1,height:300,width:600},
-  }
+    'PlayerHealth': { x: 100, y: 300, height: 200, width: 350, scale:1 },
+    'TargetHealth': { x: 600, y: 300, height: 200, width: 350,  scale:1 },
+    'Warband': { x: 100, y: 10 , scale:1, height: 200, width: 350 },
+    'Chat': { x: 10, y: 100, scale:1, height: 300, width: 600 },
+  };
```

Comment:
use camel case for object(var) names

https://github.com/airbnb/javascript#naming--camelCase